doc/contributing.md

```diff
@@ -165,6 +165,8 @@ To increase the chances of getting your pull request accepted quickly, try to:
     [Testing plots](#testing-plots).
   - Include an example of new features in the gallery or tutorials.
     Please refer to [Gallery plots](#gallery-plots) or [Tutorials](#tutorials).
+    If adding a new method/function/class, the gallery example or tutorial can
```

Comment:
```suggestion
    If adding a new method/function/class, the gallery example or tutorial should
```
I think it's better to suggest having the module examples in a different PR than the one that wraps them (which I think is the intent of previous guidelines).